app/views/posts/index.html.erb

```diff
@@ -1 +1,20 @@
-<h1>Here is a list of posts for a given user</h1>
\ No newline at end of file
+<div class="container">
+    <h1>Posts List</h1>
+    <%= render partial: "/shared/user", locals: { user: @user } %>
+    <div class="user-posts">
+        <% @posts.each.with_index(1) do |post, index| %>
+            <div class="post-with-comments">
+            <a href="/users<%= @user.id %>/posts<%= post.id %>">
+                <%= render partial: "/shared/post", locals: { post: post, index: index } %>
+            </a>
+                <div class="comments">
+                    <% post.comments.order(created_at: :desc).limit(5).each do |comment| %>
+                        <%= render partial: "/shared/comment", locals: { comment: comment } %>
+                    <% end %>
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.), Please use the `five_most_recent_comments_for_post` method that you created in the `post` model instead of what you did here.

https://github.com/JonahKayizzi/Blog-App-Ruby-on-Rails/blob/a3ec8b7906f14504dafd59a0282add67e26a7d06/app/models/post.rb#L16-L18

Reply:
Great. Thanks for the guidance